[MTA] [RN] Release notes for MTA 8.1.1 (#347)
migtools/mta-documentation: docs/topics/release-notes-topics/ref_fixed-issues-8-1-1.adoc

:_newdoc-version: 2.18.7
:_template-generated: 2026-04-15
:_mod-docs-content-type: REFERENCE

[id="fixed-issues-8-1-1_{context}"]

> = Fixed issues

Author: Hi @sjd78, could you please review the following release notes?
Thank you!

Reviewer: LGTM

[role="_abstract"]
{ProductFirstRef} ({ProductShortName}) version 8.1.1 provides the following fixed issues that have a significant impact.

The Application discovery manifest in the {ProductShortName} UI displays YAML-formatted content::
Before this update, the Application discovery manifest in the {ProductFullName} user interface (UI) did not display YAML-formatted content. As a consequence, you could only see raw JSON data that included all database record metadata instead of formatted YAML. With this release, the Application discovery manifest includes only YAML-formatted content fields.
+
(link:https://redhat.atlassian.net/browse/MTA-6774[MTA-6774])

> The {ProductShortName} UI generates assets for applications without a discovery manifest::

Author: Just a bit unsure whether it is a fix or an enhancement

Reviewer: LGTM.  It's a fix

> Before this update, you could not generate assets for applications without an associated discovery manifest by using the {ProductFullName} user interface (UI). As a consequence, you could not generate assets for applications with only source code that had not been discovered in a source platform. With this update, {ProductShortName} supports generating assets for applications without a discovery manifest. As a result, you can use the {ProductShortName} UI to generate these assets, which enhances deployment flexibility.

Reviewer 2: ```
Before this update, you could not generate assets for applications without an associated discovery manifest by using the {ProductFullName} user interface (UI). As a consequence, you were unable to generate assets for applications with only source code that had not been discovered in a source platform. With this update, {ProductShortName} supports generating assets for applications without a discovery manifest. As a result, you can use the {ProductShortName} UI to generate these assets, which enhances deployment flexibility.
```

you have already used "could not" phrasing in the previous sentences. Should you consider replacing it to break the repitition?

not a blocker, only a style suggestion.

Author: @anarnold97, now that you pointed this out, I think we should really break the repetition. Here's my suggestion:
```suggestion
Before this update, the {ProductFullName} user interface (UI) did not allow you to generate assets for applications without an associated discovery manifest. As a consequence, you could not generate assets for applications with only source code that had not been discovered in a source platform. With this update, {ProductShortName} supports generating assets for applications without a discovery manifest. As a result, you can use the {ProductShortName} UI to generate these assets, which enhances deployment flexibility.
```
WDYT?

Reviewer 2: that reads really nicely
